tests/providers/test_person.py

```diff
@@ -251,83 +252,95 @@ def test_name_female(self):
         assert last_name in CsCZProvider.last_names_female
 
 
-class TestZhCN(unittest.TestCase):
+class TestZh(unittest.TestCase):
```

Comment:
I'd rather have separate TestCases. It makes it easier to maintain.

Reply:
Can you split the test cases in `TestZhCN` and `TestZhTW`? I realize it would result in duplicated code, but it will make it easier to maintain if the two locales will ever diverge in the future.

Reply:
ok